ldapauthenticator/ldapauthenticator.py

```diff
@@ -161,7 +249,7 @@ def getConnection(userdn, username, password):
                     username=username,
                     userdn=userdn
             ))
-            conn = ldap3.Connection(server, user=userdn, password=password)
+            conn = ldap3.Connection(server, user=escape_filter_chars(userdn), password=password)
```

Comment:
Escaping filter chars from userdn can make the valid userdn invalid, e.g. escaping "(" or ")" which are valid characters for this parameter.